README.md

```diff
@@ -1,7 +1,7 @@
 # Runbook
 
 [![Gem Version](https://badge.fury.io/rb/runbook.svg)](https://badge.fury.io/rb/runbook)
-[![Build Status](https://travis-ci.org/braintree/runbook.svg?branch=master)](https://travis-ci.org/braintree/runbook)
+[![Build Status](https://github.com/braintree/runbook/actions/workflows/ruby.yml/badge.svg?branch=master&event=push)](https://github.com/braintree/runbook/actions?query=branch%3Amaster+event%3Apush)
 [![Gitter](https://badges.gitter.im/braintree/runbook.svg)](https://gitter.im/braintree/runbook?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```

Comment:
Should we ditch this also as long as we are in here? It looks like no action in over a year and Patrick is gone now.

Reply:
I removed this thinking that GitHub issues can fill the gap